### ISSUE TITLE
Advice about parameters

### ISSUE THREAD

**Issue by Vlad15lav** [zylo117/Yet-Another-EfficientDet-Pytorch#572]
@zylo117 Hi, thanks for the repository. I use the same dataset #569, but all 20 classes. I have already trained this dataset on Faster R-CNN, SSD, YOLOv3 and there was a good result (65-75) mAP, but on this model 43 mAP. 
link: https://arxiv.org/ftp/arxiv/papers/1909/1909.00133.pdf

I used the K-means with parameters anchor_base_scale = 4, anchor_stride = 8.
```
Accuracy: 48.42%
Boxes:
 [[ 36.48  37.44]
 [ 13.44  13.44]
 [142.08 159.36]]
computed paras:  ([1.1399999999999997, 0.41999999999999993, 4.440000000000001], [(1, 1.026315789473686), (1, 1.0), (1, 1.1216216216216213)])
```

I watched the tutorial #569 and tried learning in two stages.
python train.py -c 2 -p dior --head_only True --lr 5e-4 --batch_size 4 --load_weights weights/efficientdet-d2.pth --num_epochs 100 --save_interval 1100
![image](https://user-images.githubusercontent.com/62297940/103073147-291dbf80-45d8-11eb-922d-961228cffd4b.png)

python train.py -c 2 -p dior --head_only False --lr 1e-4 --batch_size 4 --load_weights last --num_epochs 100 --save_interval 1100
![image](https://user-images.githubusercontent.com/62297940/103073159-3470eb00-45d8-11eb-8724-e7dbb42e0da2.png)

What can I do to improve the mAP?

**Comment by Vlad15lav** [comment]
@zylo117 I have 9 anchors  (accuracy 69) for Yolov3, can I install them on this repository? If I use K means for 3 clusters then I get a bad result (accuracy 47).

**Comment by zylo117** [comment]
Try creating your own anchors with this. It works for me.
https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/issues/308
